src/content/docs/solvro/Structure.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Struktura
+sidebar:
+  order: 3
+description: Struktura organizacyjna KN Solvro
+---
+
+import {
+  Steps,
+  Card,
+  CardGrid,
+  LinkCard,
+  Aside,
+} from "@astrojs/starlight/components";
+
+## Koncept
+
+Koło Naukowe Solvro zrzesza obecnie ponad 100 aktywnych członków. Wraz z dynamicznym rozwojem organizacji pojawiło się kluczowe pytanie: jak skutecznie zarządzać tak licznym zespołem i równocześnie dbać o rozwój zarówno umiejętności miękkich, jak i twardych?
+
+Na przestrzeni lat testowaliśmy różne rozwiązania organizacyjne, wyciągając z nich cenne wnioski. Ostatecznie postawiliśmy na strukturę macierzową, która pozwala każdemu członkowi działać jednocześnie w sekcji (gdzie pod opieką doświadczonego przewodniczącego rozwija umiejętności techniczne) oraz w projekcie, w którym te umiejętności przekłada się na praktykę i łączy z doskonaleniem kompetencji miękkich. Co więcej, w każdym projekcie może funkcjonować wiele zespołów z różnych sekcji, dzięki czemu efektywna współpraca i realizacja ambitnych celów stoją otworem przed każdym członkiem Koła.
+
+![](../../../assets/solvro/organizational-structure.png)
+
+<Card title="Aktywny członek" icon="openCollective">
+  Aktywnym członkiem KN Solvro jest każda osoba, która w danym semestrze
+  zrealizowała co najmniej jedno zadanie operacyjne wyznaczone przez
+  przewodniczącego sekcji lub Product Ownera. Aktywni członkowie mają prawo
+  głosu podczas walnych zgromadzeń, posiadają dostęp do zasobów koła, mogą
+  uczestniczyć w spotkaniach i wyjazdach. Jeśli obowiązek ten nie zostanie
+  spełniony, po zakończeniu semestru dana osoba jest usuwana ze struktur i
+  zasobów Koła. Wyjątek stanowią członkowie honorowi, którzy – mimo że zachowują
+  dostęp do zasobów KN Solvro i mogą uczestniczyć w spotkaniach oraz walnych
+  zgromadzeniach to nie posiadają prawa do głosowania.
+</Card>
+
+## Sekcje
+
+Każda sekcja posiada swojego przewodniczącego, który posiadając znaczne doświadczone przewodzi działaniom dziedzinowym. Jest w ciągłym kontakcie z liderami zespołów wchodzącymi w skład projektów, a wraz z innymi przewodniczącymi i zarządem wypełnia strategię KN Solvro. Przewodniczący sekcji są powoływani i odwoływani prze zarząd głosem większości.
+
+<Card title="Obowiązki i uprawnienia sekcji" icon="starlight">
+- Organizacja wyjazdów na konferencje tematyczne
+- Organizacja wyjazdów na wyjazdy tematyczne
+- Organizacja szkoleń dla członków sekcji
+- Organizacja tematycznych Solvo Talków
+- Organizacja integracji dla członków sekcji
+- Planowanie kwartalnych celów operacyjnych
+- Działania promocyjne sekcji
+- Dokumentacja działań sekcji
+- Pozyskiwanie partnerów
+- Headhunting nowych członków
+- Zarządzanie kanałami komunikacji sekcji
+- Współpraca z innymi sekcjami
+
+</Card>
+
+// prędzej czy później każda sekcja będzie miała swoją stronę, w której będzie więcej opisane
```

Comment:
nie rób na zaś, napisz sekcje na razie jako zwykły tekst, zostawianie zepsutych linków i przycisków obniża zaufanie do dokumentacji, jak nie ma jakiejś strony to nie twórz linka do niej "bo kiedyś będzie"

Reply:
ewentualnie napisz bardzo wyraźnie, że jest to TODO, żeby ludzie nie klikali bez sensu